docker-entrypoint.sh

```diff
@@ -205,11 +205,21 @@ docker_process_sql() {
 docker_setup_db() {
 	# Load timezone info into database
 	if [ -z "$MYSQL_INITDB_SKIP_TZINFO" ]; then
+		# Aria in 10.4+ is slow due to "transactional" (crash safety)
+		# https://jira.mariadb.org/browse/MDEV-23326 / https://github.com/docker-library/mariadb/issues/262
+		for table in time_zone time_zone_leap_second time_zone_name time_zone_transition time_zone_transition_type
+		do
+			echo "/*!100400 ALTER TABLE $table TRANSACTIONAL=0 */;"
+		done| docker_process_sql --dont-use-mysql-root-password --database=mysql
 		# sed is for https://bugs.mysql.com/bug.php?id=20545
 		mysql_tzinfo_to_sql /usr/share/zoneinfo \
 			| sed 's/Local time zone must be set--see zic manual page/FCTY/' \
 			| docker_process_sql --dont-use-mysql-root-password --database=mysql
 			# tell docker_process_sql to not use MYSQL_ROOT_PASSWORD since it is not set yet
+		for table in time_zone time_zone_leap_second time_zone_name time_zone_transition time_zone_transition_type
+		do
+			echo "/*!100400 ALTER TABLE $table TRANSACTIONAL=1 */;"
+		done| docker_process_sql --dont-use-mysql-root-password --database=mysql
```

Comment:
This is good, but I think it would be better if we cut down on the number of calls to `docker_process_sql` and use a bit more whitespace so it's easier to read:

```suggestion
		{
			# Aria in 10.4+ is slow due to "transactional" (crash safety)
			# https://jira.mariadb.org/browse/MDEV-23326
			# https://github.com/docker-library/mariadb/issues/262
			local tztables=( time_zone time_zone_leap_second time_zone_name time_zone_transition time_zone_transition_type )
			for table in "${tztables[@]}"; do
				echo "/*!100400 ALTER TABLE $table TRANSACTIONAL=0 */;"
			done

			# sed is for https://bugs.mysql.com/bug.php?id=20545
			mysql_tzinfo_to_sql /usr/share/zoneinfo \
				| sed 's/Local time zone must be set--see zic manual page/FCTY/'

			for table in "${tztables[@]}"; do
				echo "/*!100400 ALTER TABLE $table TRANSACTIONAL=1 */;"
			done
		} | docker_process_sql --dont-use-mysql-root-password --database=mysql
		# tell docker_process_sql to not use MYSQL_ROOT_PASSWORD since it is not set yet
```